src/azure-cli-core/azure/cli/core/test_utils/vcr_test_base.py

```diff
@@ -355,6 +371,8 @@ def _execute_live_or_recording(self):
             if callable(tear_down) and not self.skip_teardown:
                 self.tear_down()
 
+    @mock.patch('azure.cli.command_modules.datalake.analytics.custom._get_uuid_str',
+                _mock_get_uuid_str)
```

Comment:
@derekbekoe @tjprescott is it possible for me to override the existing _execute_playback in my test class which will call super and then include the mock.patch there? Or would that not work?